### PR TITLE
Feature/1194 comment editing

### PR DESCRIPTION
**Pull request by chris-lawton**

### CODE REVIEW

File: opentech/apply/funds/templates/funds/includes/activity-feed.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h4 class="activity-feed__heading">Activity Feed <svg class="icon icon--speech-b
 
     <div class="wrapper wrapper--medium wrapper--activity-feed js-tabs-content">
         <div class="tabs__content tabs__content--current" id="tab-1">
-            {% include "activity/include/comment_list.html" with submission_title=True %}
+            {% include "activity/include/comment_list.html" with submission_title=True editable=False %}
```
Comment (todd-dembrey) [inline opentech/apply/funds/templates/funds/includes/activity-feed.html]: `editable=False` not necessarily required as will be falsey without this. But happy to include

File: opentech/static_src/src/javascript/apply/edit-comment.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            headers: {
+                'Content-Type': 'application/json',
+                'X-CSRFToken': $.cookie('csrftoken')
+            },
+            body: JSON.stringify({
+                message: editedComment
+            })
+        }).then(response => {
+            if (response.status === 404) {
+                $(this).closest(editBlock).append('<p class="wrapper--error js-comment-error"><b>Update unsuccessful</b>. This comment has been edited elsewhere - please refresh to get the latest updates.</p>');
```
Comment (todd-dembrey) [inline opentech/static_src/src/javascript/apply/edit-comment.js]: Do we want to note that they will lose unsaved changes by doing this?

File: opentech/static_src/src/javascript/apply/edit-comment.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                $(this).attr('disabled', true);
+                return;
+            }
+            return response.json();
```
Comment (todd-dembrey) [inline opentech/static_src/src/javascript/apply/edit-comment.js]: Do we need to handle 500 or catch generally? network issues etc?

File: opentech/static_src/src/javascript/apply/edit-comment.js (hunk 2 above)
Comment (chris-lawton) [reply opentech/static_src/src/javascript/apply/edit-comment.js]: Good point, have updated.

File: opentech/static_src/src/javascript/apply/edit-comment.js (hunk 3 above)
Comment (chris-lawton) [reply opentech/static_src/src/javascript/apply/edit-comment.js]: Definitely. I've caught all other non-404 errors.

**Pull request merged**